### PR TITLE
Chore: Release paseo runtime @ polkadot v1.2.5

### PR DESCRIPTION
**Pull request by al3mart**

### CODE REVIEW

**Review by hbulgarini — APPROVED** [review]

File: relay/paseo/build.rs [hunk 1]
```diff
@@ -6,20 +6,25 @@
...
+}
 
+#[cfg(all(feature = "std", feature = "metadata-hash"))]
 fn main() {
-	WasmBuilder::new()
-		.with_current_project()
-		.import_memory()
-		.export_heap_base()
+	substrate_wasm_builder::WasmBuilder::init_with_defaults()
+		.enable_metadata_hash("DOT", 10)
```
Comment (carlosala) [inline relay/paseo/build.rs:25]: According to `system.properties` this should be `"PAS", 10`.
```suggestion
		.enable_metadata_hash("PAS", 10)
```

Comment (hbulgarini) [reply relay/paseo/build.rs]: thank you man!

File: system-parachains/bridge-hub-paseo/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,1021 @@
...
+		pub aura: Aura,
+	}
+}
+
+#[sp_version::runtime_version]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+    spec_name: create_runtime_str!("bridge-hub-polkadot"),
+    impl_name: create_runtime_str!("bridge-hub-polkadot"),
+    authoring_version: 1,
+    spec_version: 1_001_002,
```
Comment (hbulgarini) [inline system-parachains/bridge-hub-paseo/src/lib.rs:165]: @claravanstaden i have just realized about this. Did you forget upgrading the spec version to 1_001_005 or the code is effectively running under 1_001_002?

**Review by hbulgarini — APPROVED** [review]
IMHO, the PR is in a good position to be merged.

**Pull request merged**